apps/omg_eth/lib/eth.ex

```diff
@@ -51,6 +51,31 @@ defmodule OMG.Eth do
   @spec syncing?() :: boolean
   def syncing?, do: node_ready() != :ok
 
+  @doc """
+  Send transaction to be singed by a key managed by Ethereum node, geth or parity
+  """
+  @spec send_transaction(map()) :: {:ok, hash()} | {:error, any()}
+  def send_transaction(txmap, opts \\ []) do
```

Comment:
please document the dual way the passphrase is fetched for parity, and the fact that it is not fetched/necessary at all for geth